contributors/guide/first-contribution.md

```diff
@@ -93,10 +137,36 @@ These are located in the SIG-specific community directories:
 ### File an Issue
 
 Not ready to contribute code, but see something that needs work?
-While the community encourages everyone to contribute code, it is also appreciated when someone reports an issue (aka problem).
+While the community encourages everyone to contribute code, it is also appreciated 
+when someone reports an issue.
 Issues should be filed under the appropriate Kubernetes subrepository.
-Check the [issue triage guide](./issue-triage.md) for more information.
+Check the [issue triage guide] for more information.
+
+For example, a documentation issue should be opened in [kubernetes/website].
+
+Make sure to adhere to the prompted submission guidelines while opening an issue.
+
+[Contributor Role Board]: https://discuss.kubernetes.io/c/contributors/role-board
+[k8s-ci-robot]: https://github.com/k8s-ci-robot
+[Non-Code Contributions Guide]: ./non-code-contributions.md
+[multiple repositories]: https://github.com/kubernetes/
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[help wanted]: https://go.k8s.io/help-wanted
+[good first issue]: https://go.k8s.io/good-first-issue
+[extra assistance]:./help-wanted.md
+[sl]: /sig-list.md
+[video meetings]: https://kubernetes.io/community/
+[sig-contributor-experience]: /sig-contributor-experience/README.md
+[bi-weekly meetings]: https://docs.google.com/document/d/1qf-02B7EOrItQgwXFxgqZ5qjW0mtfu5qkYIF1Hl4ZLI/edit
+[container networking interface]: https://github.com/containernetworking/cni
+[network SIG]: http://git.k8s.io/community/sig-network
+[here]: /sig-contributor-experience#github-teams
```

Comment:
```suggestion
[SIG contributor-experience]: /sig-contributor-experience#contact
```
I double checked the link, GitHub teams isn't actually a linkable item anymore -_- updated it to the right section.